pages/dkp/kaptain/2.1.0/user-management/identity-providers/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: layout.pug
+navigationTitle: Identity providers
+title: Authenticate users using an identity provider
+menuWeight: 10
+excerpt: Authenticate users to access Kaptain with an identity provider
+beta: false
+enterprise: false
+---
+
+## Prerequisites
+
+In Kaptain, authenticating users and user groups is carried out by DKP’s Dex instance. To use an alternative external identity provider as an authenticator, you can set up [DKP’s Dex to use the identity provider of your choice][ident].
+
+## Setup
+
+When you install Kaptain in your **management cluster**, the identity provider of your choice is already integrated with your DKP Dex instance, so **no further steps** are required for you to be able to enable access to your Kaptain instance with the credentials defined with your identity provider.
+
+When Kaptain is installed in a **managed cluster**, ensure the managed cluster can communicate with the Dex instance in the management cluster. For this, configure [Kaptain to authenticate with a DKP management cluster via Dex][dex].
+
+## Access Kaptain with your Identity provider credentials
+
+<p class="message--note"><strong>NOTE: </strong>By default, your identity provider users and user groups have access to the dashboard. To add or remove groups, alter the <b>Allow List</b> via the <b>Configuration service</b>.</p>
+
+1.  Open the [log-in page](../../install/deploy-kaptain#log-in-to-kaptain-using-the-management-clusters-dex-instance) to access Kubeflow’s dashboard of Kaptain.
+
+1.  Click **Log in with LDAP**.
```

Comment:
```suggestion
1.  Select **Log in with LDAP**.
```

Reply:
This dialog just displays whatever the user configured in the "displayName" field of the Dex Connector.    For example, I have a test LDAP connector that has this as the displayName:

apiVersion: dex.mesosphere.io/v1alpha1
kind: Connector
...
  displayName: LDAP connector test

And so then the login dialog says "Login with LDAP connector test". 

In short, there's no way to predict what this dialog box will say.  If we implemented this in our internal clusters it would probably say "Login with OneLogin" 

So I think the instruction here needs to be made more vague.   Select "Login with _your identity provider_" or something like that.

Reply:
```suggestion
1.  Select **Log in with *your identity provider***.
```